test/k8sT/Services.go

```diff
@@ -1362,6 +1364,29 @@ var _ = Describe("K8sServicesTest", func() {
 						testExternalIPs()
 					})
 
+					Context("With host policy", func() {
+						var ccnpHostPolicy string
+
+						BeforeAll(func() {
+							ccnpHostPolicy = helpers.ManifestGet(kubectl.BasePath(), "ccnp-host-policy-nodeport-tests.yaml")
+							_, err := kubectl.CiliumPolicyAction(helpers.DefaultNamespace, ccnpHostPolicy,
+								helpers.KubectlApply, helpers.HelperTimeout)
+							Expect(err).Should(BeNil(),
+								"Policy %s cannot be applied", ccnpHostPolicy)
+						})
+
+						AfterAll(func() {
+							_, err := kubectl.CiliumPolicyAction(helpers.DefaultNamespace, ccnpHostPolicy,
+								helpers.KubectlDelete, helpers.HelperTimeout)
+							Expect(err).Should(BeNil(),
+								"Policy %s cannot be deleted", ccnpHostPolicy)
+						})
+
+						It("Tests NodePort", func() {
+							testNodePort(true, false, helpers.ExistNodeWithoutCilium(), 0)
```

Comment:
@brb Is there any other service-related test that would be worth executing here? I.e., are there packet paths that are not tested by `testNodePort`?

Reply:
`testNodePort` (from outside) should be enough. Are you interested in testing the host-fw from inside a cluster?

Reply:
Yep, from inside the cluster would be useful as well. I was under the impression `testNodePort` did that already. Are there some cases it doesn't test?